.github/workflows/ci.yml

```diff
@@ -50,6 +55,19 @@ jobs:
       - name: Print Runner IP
         run: echo $MAIN_IP
 
+      - name: Generate TLS Certs
+        run: ./.github/workflows/scripts/tls.sh
+        env:
+          MAIN_IP: ${{ env.MAIN_IP }}
+
+      - name: Set CA certificate
+        run: |
+          {
+            echo 'TLS_CA_CERT<<EOF'
+            cat step/ca.crt
+            echo EOF
+          } >> "$GITHUB_ENV"
```

Comment:
This was annoying to figure out. This syntax allows us to output a multi line file to an environment variable. This environment variable will be used to configure the new TLS option on the action, similar to using a secret.